packages/fluent/scss/core/_variables.scss

```diff
@@ -95,10 +91,12 @@ $kendo-border-radius-lg: ( $kendo-border-radius * 2 ) !default;
 // Metrics
 $kendo-padding-x: map.get( $kendo-spacing, 2 ) !default;
 $kendo-padding-y: map.get( $kendo-spacing, 1 ) !default;
-$kendo-padding-x-sm: math.div( $kendo-padding-x, 2) !default;
-$kendo-padding-y-sm: math.div( $kendo-padding-y, 2) !default;
-$kendo-padding-x-lg: ( $kendo-padding-x * 1.5) !default;
-$kendo-padding-y-lg: ( $kendo-padding-y * 1.5) !default;
+$kendo-padding-sm-x: map.get( $kendo-spacing, 1 ) !default;
+$kendo-padding-sm-y: map.get( $kendo-spacing, 0.5 ) !default;
+$kendo-padding-md-x: map.get( $kendo-spacing, 2 ) !default;
+$kendo-padding-md-y: map.get( $kendo-spacing, 1 ) !default;
```

Comment:
```suggestion
$kendo-padding-md-x: $kendo-padding-x !default;
$kendo-padding-md-y: $kendo-padding-y !default;
```

Reply:
The base padding variables are:

1) Unreliable -- they dont really make sense, as each theme is unique and cannot be properly transferred from one theme to another
2) Inconsistent -- sometimes, to achieve a certain fixed size, we multiply and divide and mix and match due to said uniqueness
3) Obsolete, as we are moving towards spacing maps